bokeh/_testing/plugins/file_server.py

```diff
@@ -118,7 +118,7 @@ def where_is(self, path: str) -> str:
 @pytest.fixture(scope='session')
 def file_server(request: pytest.FixtureRequest) -> SimpleWebServer:
     server = SimpleWebServer()
-    server.start()
+    server.install()
```

Comment:
This is the `start` method of a Python built-in `SimpleWebServer`, not a Bokeh `Server` object (change should be reverted)